src/tests/ftest/pool/svc.py

```diff
@@ -141,23 +145,23 @@ def test_pool_svc(self):
                 pool_leader = self.check_leader(pool_leader, True)
                 non_leader_ranks.remove(pool_leader)
 
-                # Stop a pool non-leader
-                non_leader = non_leader_ranks[-1]
-                self.log.info(
-                    "Stopping a pool non-leader (%s): %s",
-                    non_leader_ranks, non_leader)
-                try:
-                    self.server_managers[-1].stop_ranks(
-                        [non_leader], self.test_log)
-                except TestFail as error:
-                    self.log.info(error)
-                    self.fail(
-                        "Error stopping a pool non-leader - "
-                        "DaosServerManager.stop_ranks([{}])".format(non_leader))
-
-                self.pool.wait_for_rebuild(to_start=True, interval=1)
-                self.pool.wait_for_rebuild(to_start=False, interval=1)
-                # Verify the pool leader has not changed
-                self.check_leader(pool_leader, False)
+                if svc_params[1] == 5:
```

Comment:
Just for robustness, you might consider
```suggestion
                if svc_params[1] >= 5:
```